app/components/Form/ApplicationForm.tsx

```diff
@@ -325,27 +301,29 @@ const ApplicationForm: React.FC<Props> = ({
   };
 
   const handleChange = (e: IChangeEvent<any>) => {
+    if (isAcknowledgementPage) updateAreAllAcknowledgementFieldsSet(e.formData);
+    if (isSubmitPage) updateAreAllSubmissionFieldsSet(e.formData);
```

Comment:
Same as for the acknowledgement page, there is probably a way to enforce that all submission fields are set via the JSON schema spec.

Reply:
This could be done with a `oneOf`: either fields are required, or the review page checkbox is checked. Doing it this way would also mean that the only thing we're ignoring is the "required" attribute, and are still validating data types